modules/multi-architecture-upgrade-mirrors.adoc

```diff
@@ -7,7 +7,7 @@
 
 = Upgrading a cluster with multi-architecture compute machines
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a cluster that supports multi-architecture compute machines.
+To upgrade your cluster with multi-architecture compute machines from {product-title} version {product-version} to a version 4.12 cluster, you must use the `candidate-4.12` update channel. For more information, see "Understanding upgrade channels". If you want to  upgrade your cluster to a specific {product-version} z-stream version, you can perform the following explicit upgrade command. 
```

Comment:
@Prashanth684 Hey Prashanth! In this case should the `candidate-4.12` be `candidate-4.11`? 

 I think the `candidate-4.12` is correct becuase you are upgrading to 4.12, but if you go to the upgrade section it brings you to the 4.11 links. Which is kinda fine, but I also just wasn't sure.

Reply:
the channel should be `candidate-4.12`, yes